music21/roman.py

```diff
@@ -1606,13 +1606,20 @@ class RomanNumeral(harmony.Harmony):
     >>> cp(r)
     ['A-5', 'C6', 'E-6', 'G-6']
 
-    >>> r = roman.RomanNumeral('VId7')
+    >>> r = roman.RomanNumeral('VId2')
     >>> r.figure
-    'VId7'
+    'VId2'
```

Comment:
this should be VId42 -- music21 prefers and emphasizes the 42 rather than 2 for 3rd inversion.

This should also have text such as "inversions of dominant 7ths can be created by specifying 'd' followed by the figures"